### ISSUE TITLE
MXNet backend does not support scale/center param in BatchNormalization equal to False

### ISSUE THREAD

**Issue by roywei** [awslabs/keras-apache-mxnet#136]
Adding BatchNormalization layer with scale or center to False will fail:
``` 
BatchNormalization(scale=False)
```
Applies to any model use BatchNorm.

Reproducible code:
change example cifar10_resnet.py, in resnet_layer function, change Batchnorm to use scale=False or center=False.
Change line 149 or 154 here:
https://github.com/awslabs/keras-apache-mxnet/blob/332c62f7b521b109431c0ef91317b2ce00e4cfb7/examples/cifar10_resnet.py#L149

to
```
x = BatchNormalization(scale=False)(x)
```
or 
```
x = BatchNormalization(center=False)(x)
```

Error Message:
```
Traceback (most recent call last):
  File "/Users/lawei/Documents/Workspace/roywei/keras/examples/cifar10_resnet.py", line 338, in <module>
    metrics=['accuracy'])
  File "/Users/lawei/Documents/Workspace/roywei/keras/keras/backend/mxnet_backend.py", line 4715, in compile
    fixed_param_names=self._fixed_weights)
  File "/Users/lawei/Documents/Workspace/roywei/incubator-mxnet/python/mxnet/module/bucketing_module.py", line 84, in __init__
    _check_input_names(symbol, fixed_param_names, "fixed_param", True)
  File "/Users/lawei/Documents/Workspace/roywei/incubator-mxnet/python/mxnet/module/base_module.py", line 52, in _check_input_names
    raise ValueError(msg)
ValueError: You created Module with Module(..., fixed_param_names=['batchnorm8_gamma', 'batchnorm6_gamma', 'batchnorm22_gamma', 'batchnorm16_gamma', 'batchnorm4_gamma', 'batchnorm0_gamma', 'batchnorm34_gamma', 'batchnorm26_gamma', 'batchnorm14_gamma', 'batchnorm24_gamma', 'batchnorm32_gamma', 'batchnorm36_gamma', 'batchnorm12_gamma', 'batchnorm18_gamma', 'batchnorm20_gamma', 'batchnorm2_gamma', 'batchnorm30_gamma', 'batchnorm28_gamma', 'batchnorm10_gamma']) but input with name 'batchnorm8_gamma' is not found in symbol.list_arguments(). Did you mean one of:
	/input_11
	conv2d_1/kernel1
	conv2d_1/bias1
	batch_normalization_1/beta1
	conv2d_2/kernel1
	conv2d_2/bias1
	batch_normalization_2/beta1
	conv2d_3/kernel1
	conv2d_3/bias1
	batch_normalization_3/beta1
	conv2d_4/kernel1
	conv2d_4/bias1
	batch_normalization_4/beta1
	conv2d_5/kernel1
	conv2d_5/bias1
	batch_normalization_5/beta1
	conv2d_6/kernel1
	conv2d_6/bias1
	batch_normalization_6/beta1
	conv2d_7/kernel1
	conv2d_7/bias1
	batch_normalization_7/beta1
	conv2d_10/kernel1
	conv2d_10/bias1
	conv2d_8/kernel1
	conv2d_8/bias1
	batch_normalization_8/beta1
	conv2d_9/kernel1
	conv2d_9/bias1
	batch_normalization_9/beta1
	conv2d_11/kernel1
	conv2d_11/bias1
	batch_normalization_10/beta1
	conv2d_12/kernel1
	conv2d_12/bias1
	batch_normalization_11/beta1
	conv2d_13/kernel1
	conv2d_13/bias1
	batch_normalization_12/beta1
	conv2d_14/kernel1
	conv2d_14/bias1
	batch_normalization_13/beta1
	conv2d_17/kernel1
	conv2d_17/bias1
	conv2d_15/kernel1
	conv2d_15/bias1
	batch_normalization_14/beta1
	conv2d_16/kernel1
	conv2d_16/bias1
	batch_normalization_15/beta1
	conv2d_18/kernel1
	conv2d_18/bias1
	batch_normalization_16/beta1
	conv2d_19/kernel1
	conv2d_19/bias1
	batch_normalization_17/beta1
	conv2d_20/kernel1
	conv2d_20/bias1
	batch_normalization_18/beta1
	conv2d_21/kernel1
	conv2d_21/bias1
	batch_normalization_19/beta1
	dense_1/kernel1
	dense_1/bias1
```

**Comment by roywei** [comment]
Root cause:
When scale=False, gamma is passed to mxnet backend as None, thus there is no symbol related to gamma.  however during compile in mxnet backend, gamma names are found in list_arguments, and it's been treated as fixed_args (arguments should not be updated). Then it fails in Bucketing Module when checking fixed_args names, no such symbol can be found.

Same case for center=False, and beta is passed as None.

**Comment by roywei** [comment]
#137  should fix this, pending merging back to master branch

**Comment by roywei** [comment]
fixed with #137, closing
